src/wix/WixToolset.Core/Link/AddDefaultSymbolsCommand.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information.
+
+namespace WixToolset.Core.Link
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using WixToolset.Data;
+    using WixToolset.Data.Symbols;
+
+    internal class AddDefaultSymbolsCommand
+    {
+        public static readonly string WixStandardInstallFolder = "INSTALLFOLDER";
+        public static readonly string WixStandardInstallFolderParent = "ProgramFiles6432Folder";
+        public static readonly string WixStandardInstallFolderReference = "Directory:INSTALLFOLDER";
+
+        public AddDefaultSymbolsCommand(FindEntrySectionAndLoadSymbolsCommand find, List<IntermediateSection> sections)
+        {
+            this.Find = find;
+            this.Sections = sections;
+        }
+
+        public IntermediateSection EntrySection { get; }
```

Comment:
This `EntrySection` is not used since the FindEntrySectionAndLoadSymbolsCommand was passed in.